src/style.css

```diff
@@ -0,0 +1,75 @@
+@font-face {
+  font-family: "Avenir";
+  font-weight: 300;
+  font-style: normal;
+  src: url("./fonts/Avenir\ Book.ttf") format("truetype");
+}
+
+body {
+  font-family: "Avenir", sans-serif;
+}
+
+.searchbar {
+  position: relative;
+  margin: 20px 0;
+}
+
+.searchbar-input {
+  position: relative;
+  color: #3d4e61;
+  font-family: "Avenir", sans-serif;
+  background: none;
+  border: 1px solid #e1e7ed;
+  border-radius: 4px;
+  box-sizing: border-box;
+  box-shadow: 0 1px 8px rgba(61, 78, 97, 0.1);
+  width: 100%;
+}
+
+.searchbar-input:hover {
+  box-shadow: 0 3px 8px rgba(61, 78, 97, 0.2);
+}
+
+.searchbar-input:focus {
+  font-weight: 600;
+}
+
+.search-input--wrapper {
+  position: relative;
+}
+
+.search-input--wrapper::before {
+  position: absolute;
+  display: block;
+  content: "";
+  background-image: url(./images/Search.svg);
+  background-size: cover;
+}
+
+.searchbar--big .search-input--wrapper::before {
+  height: 19px;
+  width: 19px;
+  margin-top: 25px;
+  margin-left: 26px;
+}
+
+.searchbar--big .searchbar-input {
+  padding-left: 62px;
+  font-size: 16px;
+  line-height: 22px;
+  height: 70px;
+}
+
+.searchbar--small .search-input--wrapper::before {
+  height: 11px;
+  width: 11px;
+  margin-top: 15px;
+  margin-left: 13px;
+}
```

Comment:
We don't have any basic search bar, so why we should use there big and small search class names? We have search-input and search-input - small (--small). Therefore, we set all styles that are supposed to be for the "large" input for all inputs in general, and with the modifier small we change only the necessary values to make our input smaller.